static/js/common.ts

```diff
@@ -82,19 +84,29 @@ export function adjust_mac_shortcuts(key_elem_class, require_cmd_style) {
 
 // See https://zulip.readthedocs.io/en/latest/development/authentication.html#password-form-implementation
 // for design details on this feature.
-function set_password_toggle_label(password_selector, label, tippy_tooltips) {
+function set_password_toggle_label(
+    password_selector: string,
+    label: string,
+    tippy_tooltips: boolean,
+): void {
     $(password_selector).attr("aria-label", label);
     if (tippy_tooltips) {
-        if (!$(password_selector)[0]._tippy) {
+        const element: ReferenceElement = $(password_selector)[0];
+        if (!element._tippy) {
             tippy(password_selector);
         }
-        $(password_selector)[0]._tippy.setContent(label);
+
+        element._tippy?.setContent(label);
```

Comment:
This can be done without the uncertainty of optional chaining (which makes it appear as if `setContent` might not get called), since `tippy()` returns the `Instance` too:

```js
        const tippy_instance = element._tippy ?? tippy(element);
        tippy_instance.setContent(label);
```

Reply:
Used this approach in the `refactor: Use variables to get TypeScript type narrowing.` commit (plus test changes due to this).